### DISCUSSION TITLE
How to assign an activity to a project afterwards?

### DISCUSSION THREAD

**Discussion by kankadev** [kimai/kimai#4520] [Questions & Answers] [answered]
Hi,

I can't find out how to assign an activity to a project. I just can create a new activity in the project edit mask. And I can't see an option in the activity edit mask. But I want to link an existing activity with a project.

Thanks a lot.

**Comment by kevinpapst** [comment, marked as answer]
You can't, for good reasons: https://www.kimai.org/documentation/activity.html#changing-the-activity-type
